modules/api_management/main.tf

```diff
@@ -136,6 +136,22 @@ resource "azurerm_api_management_diagnostic" "apim_diagnostic" {
   }
 }
 
+resource "azurerm_monitor_smart_detector_alert_rule" "example" {
+  count = length(var.alert_rules_settings)
+
```

Comment:
I think the line breaks can be removed here

Reply:
Remove empty lines